### PR TITLE
Added message form

### PR DESCRIPTION
**Pull request by marice-romero**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,6 +48,30 @@ <h2>Connect</h2>
             <p>github: <a href="https://github.com/marice-romero">github.com/marice-romero</a></p>
         </section>
 
+        <section>
+            <h2>Leave a Message</h2>
+            <form name="leave_message">
```
Comment (minazakhary3) [inline index.html]: In the javascript file you try to reference this form by using getElementById(), while the form only has the name, which gives an error in the console and breaks functionality.
You should give the form and id attribute as well`<form name="leave_message" id="leave_message">`

File: js/index.js [hunk 2]
```diff
@@ -15,4 +15,79 @@ const footer = document.querySelector('footer');
...
+    removeButton.textContent = "Remove";
+    removeButton.setAttribute('type', 'button');
+    removeButton.addEventListener('click', () => {
+        const entry = removeButton.parentNode;
+        entry.remove();
+        messageHeader.remove();
+    })
+    newMessage.appendChild(removeButton);
+
+    const messageHeader = document.createElement('h2');
```
Comment (minazakhary3) [inline js/index.js]: You declare and initialize the messageHeader variable here but you try to use it at line 34, which again gives an error and breaks functionality. You should move this line and put it before any usages of the messageHeader variable (put it above line 34)

**Review by minazakhary3 — CHANGES_REQUESTED** [review]
Needs a couple of corrections as it's currently not working (see the comments I left on the code).
But mostly looks good, great job! Fix the changes and resubmit the assignment.

File: index.html (hunk 1 above)
Comment (marice-romero) [reply index.html]: Hi Mina! Thanks so much for checking - I'm seeing an id in my local VS Code, but it didn't upload for some reason. I might have forgotten to save it?

File: js/index.js (hunk 2 above)
Comment (marice-romero) [reply js/index.js]: I was trying to clean up my code and moved this by mistake without testing it again (facepalm) - moved this back up before line 34! 

Do I just create a new pull request now?

File: index.html (hunk 1 above)
Comment (minazakhary3) [reply index.html]: You might have not added it to the commit/made the change after you committed?

File: js/index.js (hunk 2 above)
Comment (minazakhary3) [reply js/index.js]: No need for a new pull request as you have corrected them in a second commit, but can you resubmit the assignment so I can approve it?

**Pull request merged**